ddtrace/tracer/span.go

```diff
@@ -170,6 +170,24 @@ func (s *span) setSamplingPriority(priority int, sampler samplernames.SamplerNam
 	s.setSamplingPriorityLocked(priority, sampler)
 }
 
+// LocalRootSpan returns the root span of the span's trace.
+func (s *span) LocalRootSpan() Span {
```

Comment:
In addition, let's add a little bit of detail here about what a "root span" is, since (AFAIK) until now it's been totally internal to the tracer.

I'm OK exposing it without too much worry since this is still an un-exported type, making this an undocumented API which we can change as we like.

Reply:
I renamed LocalRootSpan into Root as suggested (same for the internal method).

As of making the documentation clearer @knusbaum, I gave it a try by simply saying when I though it is valid to use (ie. when the root isn't finished). Please have a look and feel free to suggest a code change with any further details you'd prefer.